docs/modules/ROOT/pages/erc1155-metadata-uri.adoc

```diff
@@ -0,0 +1,75 @@
+= ERC-1155 Metadata URI
+
+The OpenZeppelin xref:erc1155.adoc[ERC-1155] Metadata URI extension is needed to manage and store URIs for individual tokens. This extension allows each token to have its own unique URI,
```

Comment:
Make use of this docs -> https://eips.ethereum.org/EIPS/eip-1155#metadata-extensions

Reply:
Let me know what you think, see https://github.com/OpenZeppelin/rust-contracts-stylus/pull/416/commits/cfe28f580955a2ad8a60c03cde434b7223eedef5